README.md

```diff
@@ -181,7 +111,7 @@ above and use the following option for building with nGraph:
 
         cd tensorflow
         git checkout v1.12.0
-        bazel build --config=opt --config=mkl --config=ngraph //tensorflow/tools/pip_package:build_pip_package 
+        bazel build --config=opt --config=ngraph //tensorflow/tools/pip_package:build_pip_package 
```

Comment:
Around line 192: `ngraph module` -> `nGraph module`

Reply:
`With this configuration, your TensorFlow model scripts will work without any changes.`

how about:
`With this configuration, your TensorFlow model scripts will work without any changes, ie, you do not need to add import ngraph_bridge, like option 1 and 2.`

Reply:
This line:
```
cd test
./gtest_ngtf
```

They need code formatting. Currently they show up as plain text

Reply:
The mint example model: `some MNIST data on your local machine`
This model is not directly runnable. The `python keras_sample.py` works out-of-the-box. Do we want to add that?

It tests both inference and training.

Reply:
`idential`->`identical`

Reply:
You mentioned for Mac we do not need this part: `Export the appropriate paths `
But if they do no harm, we can keep them for now I suppose?